content/en/docs/concepts/configuration/secret.md

```diff
@@ -29,23 +29,6 @@ writing secret data to nonvolatile storage.
 Secrets are similar to {{< glossary_tooltip text="ConfigMaps" term_id="configmap" >}}
 but are specifically intended to hold confidential data.
 
-{{< caution >}}
-Kubernetes Secrets are, by default, stored unencrypted in the API server's underlying data store (etcd). Anyone with API access can retrieve or modify a Secret, and so can anyone with access to etcd.
-Additionally, anyone who is authorized to create a Pod in a namespace can use that access to read any Secret in that namespace; this includes indirect access such as the ability to create a Deployment.
```

Comment:
IMO, this caution is rightly strongly worded and the replacement text is missing many important points that will be lost if they are not covered anywhere else. So we should consider keeping this paragraph as is, on this page because folks would have bookmarked this page and suddenly seeing this "caution" section lost would be a big surprise. 

Please let me if there is a reason I am missing (which is very likely the case) for removing this though.